examples/flow_box/main.rs

```diff
@@ -54,7 +54,7 @@ fn create_color_button(color: &'static str) -> gtk::Button {
     let rgba = gdk::RGBA::from_str(color).unwrap();
     drawing_area.set_draw_func(move |_, cr, _width, _height| {
         GdkCairoContextExt::set_source_rgba(cr, &rgba);
-        cr.paint().expect("Invalid cairo surface state");
```

Comment:
why ? :p

Reply:
you need to cargo update & put this back ^^

Reply:
mmmh, I see.
Will learn at some point to run cargo update every time before I start on a PR